src/parser.py

```diff
@@ -133,7 +135,30 @@ def _parse_while_statement(self) -> WhileStatement:
 
     def _parse_function_statement(self) -> FunctionStatement:
         log.info(AppType.PARSER, 'Started parsing FunctionStatement')
-        pass
+
+        # name
```

Comment:
Comments seems redundant given variable names. Suggest removing them, code is readable without.